### DISCUSSION TITLE
MinWidth not applied on UserControl

### DISCUSSION THREAD

**Discussion by FabianGolembiewski-sp** [AvaloniaUI/Avalonia#16795] [Q&A]
### Describe the bug

Create a Avalonia App, nothing special. Allow Resizng.
Use a UserControl and set MinWidth e.g. to 700.
The user can endless resize...

Operating System Windows 10 64 Bit.

### To Reproduce

Use Visual Studio Template for Avalonia (Windows).
Use a UserControl inside MainWindow (e.g. MainVew).
Set MinWidth on UserControl.

### Expected behavior

When setting MinWidth, the UserControl must not be resizable to a size smaller the MinWidth.

### Avalonia version

11.1.3

### OS

Windows

**Comment by maxkatz6** [comment]
I don't see where the bug is here.
UserControl size won't be smaller than MinWidth on resizing, but it will be cropped by the window itself.

**Comment by stevemonaco** [comment]
This may seem unexpected when you aren't familiar with the layout system, but children do not dictate their sizes to the parent. They have a `DesiredSize` which is a request, not a demand. The parent is free to ignore this if all of its children cannot be accommodated by the available size.

**Reply by FabianGolembiewski-sp** [reply]
> This may seem unexpected when you aren't familiar with the layout system, but children do not dictate their sizes to the parent. They have a `DesiredSize` which is a request, not a demand. The parent is free to ignore this if all of its children cannot be accommodated by the available size.

Is this the case? Ok, this should be documented.

**Comment by FabianGolembiewski-sp** [comment]
The Problem is that the Window can be resized to a Size smaller than 700. I would expect than MinWidth means it must **always** be larger than or equal to 700.
